spec/system/dashboards/user_visits_dashboard_spec.rb

```diff
@@ -22,7 +22,7 @@
 
       visit dashboard_path
 
-      within "main > header" do
+      within "header:nth-child(1)" do
```

Comment:
strangely - this  is failing after merging 

```

  1) Dashboard when looking at analytics counters shows the count of unspent credits
     Failure/Error:
       within "header:nth-child(1)" do
         expect(page).to have_text("1")
       end
     
     Capybara::ElementNotFound:
       Unable to find visible css "header:nth-child(1)" 
```

Reply:
I see the intent (there were two headers, one a child of body and one a child of a div (and a div, and a div)

I'm going to change this to `div > header` which seem horrible but passed the tests.

```
      header" do
        expect(page).to have_text("1\nCredits available")
      end
```

PR upcoming